pkg/broker/variables_test.go

```diff
@@ -80,3 +82,177 @@ func TestBrokerVariable_ToSchema(t *testing.T) {
 		})
 	}
 }
+
+func TestBrokerVariable_ValidateVariables(t *testing.T) {
```

Comment:
I suggest adding a case for when the schema syntax is incorrect to ensure that error is correctly handled.

Reply:
Added